core/src/main/java/org/apache/accumulo/core/rpc/UGIAssumingTransportFactory.java

```diff
@@ -46,7 +47,13 @@ public UGIAssumingTransportFactory(TTransportFactory wrapped, UserGroupInformati
   }
 
   @Override
-  public TTransport getTransport(final TTransport trans) {
-    return ugi.doAs((PrivilegedAction<TTransport>) () -> wrapped.getTransport(trans));
+  public TTransport getTransport(final TTransport trans) throws TTransportException {
+    return ugi.doAs((PrivilegedAction<TTransport>) () -> {
+      try {
+        return wrapped.getTransport(trans);
+      } catch (TTransportException e) {
+        throw new RuntimeException(e);
+      }
+    });
```

Comment:
Does this method even throw TTransportException now? It was added to the method signature, but seems like it doesn't throw, because it's always wrapped with RTE if it happens.